test/e2e_node/util.go

```diff
@@ -64,7 +64,7 @@ var kubeletAddress = flag.String("kubelet-address", "http://127.0.0.1:10255", "H
 
 var startServices = flag.Bool("start-services", true, "If true, start local node services")
 var stopServices = flag.Bool("stop-services", true, "If true, stop local node services after running tests")
-var busyboxImage = imageutils.GetE2EImage(imageutils.BusyBox)
+var busyboxImage = imageutils.GetE2EImage(imageutils.Agnhost)
```

Comment:
We should just call this agnhostImage

Reply:
agree with @saiyan86, not touching this var makes for a much smaller diff, but agnhost is based off of alpine so the var name is now misleading